src/pages/PatientListPage/index.tsx

```
@@ -15,6 +15,18 @@ const PatientListPage = () => {
   const data = result?.data ?? [];
   const isLoading = result?.isLoading ?? false;
 
+  // 추후 삭제
+  console.log(data); // 데이터 출력
+  console.log(isLoading);
+
+  const handlePressAdㅇPatient = (test: string) => {
```

ESLint

[error] 'handlePressAdㅇPatient' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)